.github/workflows/ci.yml

```diff
@@ -35,15 +35,15 @@ jobs:
           ytdl-test-set: download
           run-tests-ext: sh
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@master
```

Comment:
Using master is a bad idea imo. It would be better to pin it to `@v3` alongside adding https://github.com/ytdl-org/youtube-dl/pull/31429

Reply:
I pinned it to latest major version. Please doublecheck!